plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditor.java

```diff
@@ -4167,6 +4176,37 @@ public void onEndQuery(final DBCSession session, final SQLQueryResult result, DB
                 }
             }
         }
+        
+        private void prepareMetadataChangedNotification() {
```

Comment:
Why `prepare`? The notification is shown in this method, so `show` would suit better

Reply:
Agreed